optuna/samplers/_tpe/sampler.py

```diff
@@ -160,7 +160,7 @@ def objective(trial):
                 <https://papers.nips.cc/paper/4443-algorithms-for-hyper-parameter-optimization.pdf>`_
                 ). The weights of good trials, i.e., trials to construct `l(x)`, are computed by a
                 rule based on the hypervolume contribution proposed in the `paper of MOTPE
-                <https://dl.acm.org/doi/10.1145/3377930.3389817>`_.
+                <https://doi.org/10.1145/3377930.3389817>`_.
```

Comment:
As the first author of [this paper](https://doi.org/10.1145/3377930.3389817), the URL should be [`https://www.jair.org/index.php/jair/article/view/13188`](https://www.jair.org/index.php/jair/article/view/13188), which is the journal version of the conference paper, because this trick of weighting strategy was introduced in the journal paper and not included in the conference paper.

Reply:
Thank you for the valuable comment. I am going to update it. Is it the same case for the 82nd line?

Reply:
The 82nd line is OK because the 83rd line is the link to the journal paper, and both of them are MOTPE papers.

https://github.com/optuna/optuna/blob/6808c869771e8c09e2deba4d5a83a69048cc6bf7/optuna/samplers/_tpe/sampler.py#L79-L83

Reply:
I understand. I really appreciate your follow-up.
I updated the link.